.github/workflows/cp-ci.yml

```
@@ -0,0 +1,157 @@
+#---------------------------------------------------------------------
+# GitHub Action Workflow for Cloud-Pipeline
+# Version 1.1
+# 2022
+#---------------------------------------------------------------------
+
+name: cp-ci
+on:
+  push:
+    branches: 
+    - 'develop'
+    - 'release/**'
+    paths-ignore:
+      - ".github/workflows/**"
+      - ".README.md"
```

I think, to ignore `README.md` in all subdirectories. The following pattern is required:
```suggestion
      - "**README.md"
```